tomls/omnibus-base-mainnet-andromeda/oracles/statausdc-usdc.toml

```diff
@@ -0,0 +1,16 @@
+[clone.erc_4626_to_assets_ratio_oracle]
+source = "erc-4626-to-assets-ratio-oracle:1.0.0"
+options.vault = "<%= settings.stata_usdc_address %>" # On testnet, we provide no vault here and use the mocks deployed by this package
```

Comment:
missing target

Reply:
Looks like its resolved, but maybe we want to make the target `synthetix-mock-tokens` ?